### ISSUE TITLE
three.js polymer import file

### ISSUE THREAD

**Issue by NicolasRannou** [mrdoob/three.js#8972]
##### Description of the problem 

There is no import file for polymer.

We would need an official polymer import file to allow polymer to manage properly the THREEJS dependencies.

Polymer will only load THREEJS 1 time if loaded through the same import file. If not, it will try to load THREEJS multiple times and create issues.

I'd like to submit a pull request but I do not know where should this file be living.

In examples or part of the src/ repo?

threejs-import.html
```
<script src="//cdn.rawgit.com/mrdoob/three.js/master/build/three.min.js"></script>
```

##### Three.js version

- [x] Dev
- [x] r77
- [x] ...

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] Linux
- [ ] Android
- [ ] IOS

**Comment by mrdoob** [comment]
I don't have experience with polymer, so I don't know what any of this means...

**Comment by NicolasRannou** [comment]
Yes sorry let me detail a bit more:

To use THREEJS in a Polymer element, the recommended way to include THREEJS is via HTML Import. The HTML Import allows Polymer to detect if an import is needed multiple times accross the components of the application and if so, only load the HTML Import one time.

Let's imagine the case "User A" writes a THREEJS Polymer element, including THREEJS from his custom import file.

"User B" also writes a THREEJS Polymer element, including THREEJS from a different custom import file.

Then "User B" also wants to import the element "User A" created. In this case THREEJS will not work because Polymer will load THREEJS 2 times (1 time from "User A" import file and 1 time from "User B" import file).

To avoid this kind of collision issues, both users should import THREEJS from the same location.

My plan was just to create a three-import.html file (not sure which would be the most appropriate location in the Github repo) which only contains the following:
```
<script src="//cdn.rawgit.com/mrdoob/three.js/master/build/three.min.js"></script>
```

This way, all users willing to use THREEJS in their element will have a common place to import it from and avoid all kind of collision issues, multiple loads, etc.

Does it make more sense?

**Comment by satori99** [comment]
Isn't this a bit premature?

WebComponents are not enabled in IE or Firefox as of yet, and HTML Imports won't be supported at all by mozilla [0] anyway.

The polymer polyfill seems a little hackish to me anyway, as it rewrites so many DOM methods to return wrapped objects, and can't stop styles leaking in or out of elements properly.

The idea is neat, but it seems a bit early, especially for HTML Imports.

[0] https://developer.mozilla.org/en-US/docs/Web/Web_Components/HTML_Imports

**Comment by NicolasRannou** [comment]
Well, Polymer has been released for some times now and seems strong enough for production code, so no, I do not think it is premature.

Anyway, I created a component for ThreeJS and if at some point @mrdoob wants to make the move (i.e. include it three.js ) just let me know. The only down side is that it adds another step during the release process for @mrdoob.

https://github.com/NicolasRannou/three-import

Cheers,
